Uchu.World/Objects/GameObjects/GameObject.cs

```diff
@@ -148,12 +148,19 @@ protected GameObject()
                     this.GetComponent<LuaScriptComponent>() != default) return;
                 var scriptName = ((string) scriptNameValue).ToLower();
                 Logger.Debug($"{this} -> {scriptNameValue}");
+                //var a = false;
                 foreach (var (objectScriptName, objectScriptType) in Zone.ScriptManager.ObjectScriptTypes)
                 {
                     if (!scriptName.EndsWith(objectScriptName)) continue;
                     this.Zone.LoadObjectScript(this, objectScriptType);
+                //    a = true;
                     break;
                 }
+                /*
+                if (!a && scriptName != ""){
+                    Console.WriteLine("Did not load script: " + scriptName + " Object LOT: " + Lot.Id);
```

Comment:
Was this meant to be commited. Shouldn't really have commented out logging statements like this.

Reply:
This was something unrelated I was working on that I accidentally left in. It could potentially be a useful logging statement, should I switch it to the standard logger method?

Reply:
I would say it is useful as a `Logger.Debug` statement.